zeitschrift-fur-kunstgeschichte.csl

```diff
@@ -126,8 +134,8 @@
                           <text macro="volume"/>
                         </group>
                         <names variable="editor">
+                          <name and="text" delimiter-precedes-et-al="never" delimiter-precedes-last="never"/>
                           <label form="verb-short" suffix=" "/>
```

Comment:
Yes, the order here should change. It should be exactly the same as the corresponding part in the bibliography: https://github.com/citation-style-language/styles/pull/1874/files#diff-013ac955d080bfb24c61182d219f6de1R230 , i.e.
```
<names variable="editor">
   <label form="verb-short" suffix=" "/>
   <name and="text" delimiter-precedes-et-al="never" delimiter-precedes-last="never"/>
</names>
```